README.md

```diff
@@ -1,5 +1,369 @@
+        address owner,
+        address spender,
+        uint256 value
+    ) internal {
+        require(owner != address(0), "ERC20: approve from the zero address");
+        require(spender != address(0), "ERC20: approve to the zero address");
+
+        _allowances[owner][spender] = value;
+    }
+
+    function approve(address spender, uint256 value) public returns (bool) {
+        _approve(msg.sender, spender, value);
+        return true;
+    }
+
+    function transferFrom(
+        address _sender,
+        address recipient,
+        uint256 amount
+    ) public returns (bool) {
+        _transfer(_sender, recipient, amount);
+        _approve(
+            _sender,
+            msg.sender,
+            sub(_allowances[_sender][msg.sender], amount)
+        );
+        return true;
+    }
+
+    function increaseAllowance(address spender, uint256 addedValue)
+        public
+        returns (bool)
+    {
+        _approve(
+            msg.sender,
+            spender,
+            add(_allowances[msg.sender][spender], addedValue)
+        );
+        return true;
+    }
+
+    function decreaseAllowance(address spender, uint256 subtractedValue)
+        public
+        returns (bool)
+    {
+        _approve(
+            msg.sender,
+            spender,
+            sub(_allowances[msg.sender][spender], subtractedValue)
+        );
+        return true;
+    }
+
+    function _mint(address account, uint256 amount) internal {
+        require(account != address(0), "ERC20: mint to the zero address");
+
+        _totalSupply = add(_totalSupply, amount);
+        _balances[account] = add(_balances[account], amount);
+    }
+
+    function _burn(address account, uint256 value) internal {
+        require(account != address(0), "ERC20: burn from the zero address");
+
+        _balances[account] = sub(_balances[account], value);
+        _totalSupply = sub(_totalSupply, value);
+    }
+
+    function _burnFrom(address account, uint256 amount) internal {
+        _burn(account, amount);
+        _approve(
+            account,
+            msg.sender,
+            sub(_allowances[account][msg.sender], amount)
+        );
+    }
+
+    function name() public returns (string) {
+        return _name;
+    }
+
+    function symbol() public returns (string) {
+        return _symbol;
+    }
+
+    constructor() public {
+        _name = "ERC20TokenDemo";
+        _symbol = "ETD";
+        _mint(msg.sender, 1000);
+    }
+}
+```
+
+First, we need to compile the Solidity contract to wasm bytecode using [SOLL](https://github.com/second-state/soll):
+
+```
+soll -deploy=Normal erc20.sol
+```
+
+and then generating `erc20.hex` from `erc20.wasm`:
+
+```
+xxd -p erc20.wasm | tr -d $'\n' > erc20.hex
+```
+
+Now we could deploy the wasm bytecode to our node:
+
+```
+# Start a new shell in the same docker container
+docker exec -it ssvm bash
+(docker) ~/substrate-cli-tools/typescript/dist/ssvm.js create -p 99 -g 5000000 -c 0x$(cat erc20.hex)
```

Comment:
Add tip make sure user establish/run a ssvm-node before interactive it.

Reply:
Okay, I'll guide user execute `info.js` to make sure rpc server is running